index.html

```diff
@@ -305,6 +305,50 @@ <h5 class="about-me-heading">About Me</h5>
         </ul>
       </div>
     </section>
-    
+    <footer class="footer">
+      <h2 class="footer-text">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
+      <form action="https://formspree.io/f/xpzbopvv" method="POST">
+        <ul class="contact-section">
+          <li class="form-items">
+            <input
+              type="text"
+              name="name"
+              id="name"
+              class="name"
+              maxlength="20"
+              placeholder="Full name"
+              required
+            />
+          </li>
+          <li class="form-items">
+            <input class="email" type="email" placeholder="Email address" required />
+          </li>
+          <li class="form-items">
+            <textarea
+              class="textarea"
+              name="textarea"
+              id="textarea"
+              cols="30"
+              rows="10"
+              maxlength="500"
+              placeholder="Enter your text here"
+              required
+            >
+            </textarea>
```

Comment:
- [ ] Please remove all the spaces before closing `textarea` tag, these spaces are being used in the field as value and the placeholder is not visible.

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/165701234-2f73441c-69fe-47a7-9753-8e053dcd1feb.png)